modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -95,257 +95,257 @@ else {
-							}
-							else {
-								message =
-									'<%= UnicodeLanguageUtil.get(request, "the-selected-organization-is-associated-with-deactivated-users.-do-you-want-to-proceed-with-deleting-the-selected-organization-by-automatically-unassociating-the-deactivated-users") %>';
-							}
-
-							if (confirm(message)) {
-								<portlet:namespace />doDeleteOrganizations(
-									ids,
-									organizationsRedirect
-								);
-							}
+							message =
+								'<%= UnicodeLanguageUtil.get(request, "the-selected-organization-is-associated-with-deactivated-users.-do-you-want-to-proceed-with-deleting-the-selected-organization-by-automatically-unassociating-the-deactivated-users") %>';
+						}
+
+						if (confirm(message)) {
+							<portlet:namespace />doDeleteOrganizations(
+								ids,
+								organizationsRedirect
+							);
 						}
 					}
-				);
-			}
-			else if (
-				confirm(
-					'<%= UnicodeLanguageUtil.get(request, "are-you-sure-you-want-to-delete-this") %>'
-				)
-			) {
-				<portlet:namespace />doDeleteOrganizations(
-					ids,
-					organizationsRedirect
-				);
-			}
+				}
+			);
+		}
+		else if (
+			confirm(
+				'<%= UnicodeLanguageUtil.get(request, "are-you-sure-you-want-to-delete-this") %>'
+			)
+		) {
+			<portlet:namespace />doDeleteOrganizations(
+				ids,
+				organizationsRedirect
+			);
+		}
+	});
+}
+
+function <portlet:namespace />doDeleteOrganizations(
+	organizationIds,
+	organizationsRedirect
+) {
+	var form = document.<portlet:namespace />fm;
+
+	if (organizationsRedirect) {
+		Liferay.Util.setFormValues(form, {
+			redirect: organizationsRedirect,
 		});
 	}
 
-	function <portlet:namespace />doDeleteOrganizations(
-		organizationIds,
-		organizationsRedirect
-	) {
-		var form = document.<portlet:namespace />fm;
+	Liferay.Util.postForm(form, {
+		data: {
+			deleteOrganizationIds: organizationIds,
+			<%= Constants.CMD %>: '<%= Constants.DELETE %>',
+		},
+		url: '<portlet:actionURL name="/users_admin/edit_organization" />',
+	});
+}
 
-		if (organizationsRedirect) {
-			Liferay.Util.setFormValues(form, {
-				redirect: organizationsRedirect,
-			});
-		}
+function <portlet:namespace />getUsersCount(className, ids, status, callback) {
+	var formData = new FormData();
 
-		Liferay.Util.postForm(form, {
-			data: {
-				deleteOrganizationIds: organizationIds,
-				<%= Constants.CMD %>: '<%= Constants.DELETE %>',
-			},
-			url: '<portlet:actionURL name="/users_admin/edit_organization" />',
+	formData.append('className', className);
+	formData.append('ids', ids);
+	formData.append('status', status);
+
+	Liferay.Util.fetch(
+		'<liferay-portlet:resourceURL id="/users_admin/get_users_count" />',
+		{
+			body: formData,
+			method: 'POST',
+		}
+	)
+		.then(function (response) {
+			return response.text();
+		})
+		.then(function (response) {
```

Comment:
Every `then()` should have a `catch()`.

Reply:
I could and maybe should have added it. I usually just replace the AUI functions, and leave the rest as is but I'll replace it.